forte/ontology_specs/ag_news.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "ag_news",
+    "imports": [
+      "base_ontology.json"
+    ],
+    "definitions": [
+      {
+        "entry_name": "ft.onto.ag_news.Title",
```

Comment:
I think `Title` can be a common type that can be included in `base_ontology`.

Reply:
Sure, I'll fix that